### PR TITLE
accept data-js-libs not ending in ".js"

### LINKED ISSUE #46: data-js-libs not ending in .js
The `getJsLibraries` function filters out every library that doesn't match the regex `https?://.+\.js$`.

We have a case where we have to append a version code after the library. e.g `https://example.com/lib.js?version=1.2`.

Would you be ok with making the regex more broad or removing it all together?
Does anybody know why it was introduced in the first place?

### PR DESCRIPTION
**Pull request by fgutmann**
Closes #46

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
     "url-search-params": "0.7.1",
     "webpack": "^3.5.2",
     "webpack-dev-server": "^2.7.1",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.0.0",
+    "core-js": "^2.5.7"
```
Comment (kisenka) [inline package.json]: @fgutmann core-js@2.5.7 already ships with `@babel#runtime-corejs2`, there is no need to define it explicitly

Comment (fgutmann) [reply package.json]: There is now a transitive dependency to `core-js 3.0.0` from `@babel/preset-env@7.4.2`. The `3.0.0` version seems to have an incompatible directory structure with `2.x`.

```
>$ npm ls core-js
kotlin-playground@1.18.1
├─┬ @babel/preset-env@7.4.2
│ └─┬ core-js-compat@3.0.0
│   └── core-js@3.0.0 
├─┬ @babel/runtime-corejs2@7.4.2
│ └── core-js@2.6.5 
└─┬ babel-plugin-async-to-promises@1.0.5
  └─┬ babel-helper-hoist-variables@6.24.1
    └─┬ babel-runtime@6.26.0
      └── core-js@2.6.5 
```

I'm not sure if adding it explicitly is the best option though. I just wanted to get it building :-)

Comment (kisenka) [reply package.json]: Ah I see. We should downgrade to babel 5.3 then. Could you do it?

**Comment by fgutmann** [comment]
Sure :-)

**Comment by fgutmann** [comment]
According to the [official babel 7.4 release documentation](https://babeljs.io/blog/2019/03/19/7.4.0.html#migration-from-core-js-2), core-js 3 should not be enabled by default and 7.4 is explicitly compatible with core-js 2. However it clearly depends on core-js 3 in our case, which breaks things.

As suggested by @kisenka, downgraded to babel 7.4 and rebased both pull requests.

File: src/js-executor/index.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ class JsExecutor {
 
   async execute(jsCode, jsLibs, theme) {
     const loadedScripts = (this.iframe.contentDocument || this.iframe.document).getElementsByTagName('script').length;
-    // 2 scripts by default: INIT_SCRIPT + JQuery
+    // 2 scripts by default: INIT_SCRIPT + kotlin stdlib
```
Comment (AlexanderPrendota) [inline src/js-executor/index.js]: Hi! Why `stdlib`? We loaded 1 external lib - JQuery.

Comment (fgutmann) [reply src/js-executor/index.js]: In the `reloadIframeScripts` two scripts are hard coded. One is the `kotlin.js` file and the other is the INIT_SCRIPT. jQuery on the other hand is added in `getJsLibraries` and therefore part of the `jsLibs`.

I guess things changed at some point in time and the comment was not updated.

Comment (AlexanderPrendota) [reply src/js-executor/index.js]: Yeah. Sure, It's my fault

**Comment by AlexanderPrendota** [comment]
LGTM.
Also, I have to build examples page. I'll do it on my side after merging your branch.
@kisenka approved?

**Comment by kisenka** [comment]
@fgutmann thanks a lot! :)

**Comment by fgutmann** [comment]
Thanks for merging!

**Pull request merged**